lib/go-tc/regions.go

```diff
@@ -19,10 +19,12 @@ package tc
  * under the License.
  */
 
+// RegionsResponse ...
```

Comment:
These comments should generally be removed, I think. They suppress the linter but don't address its concern.

Reply:
We plan on circling back to all the endpoints and document them with Swagger meta tags so those are temporary